dump_zenodo_open_metadata.py

```diff
@@ -60,11 +60,11 @@ def parse_record(rec, spam=False):
 }
 
 choices = [k + ' - ' + v for k, v in DUMP_TYPES.items()]
-choice = str(input("What type of dump you want? {}".format(', '.join(choices)) + '\n'))
+choice = str(raw_input("What type of dump you want? {}".format(', '.join(choices)) + '\n'))
```

Comment:
```suggestion
choice = str(input("What type of dump you want? {}".format(', '.join(choices)) + '\n'))
```

Since we're in Python 2 :)